# Header
std_msgs/Header header
uint8 version

# Bullshit part
uint16 timestamp

int32 targetPosition
int32 targetVelocity
int16 targetTorque
int32 positionOffset
int16 torqueOffset
int32 velocityOffset
uint32 profileAcceleration
uint32 profileDeceleration
uint16 motionProfileType
uint8 operationMode